Suggested changes for 5.5.0 changelog (#6045)
tinymce/tinymce: modules/tinymce/changelog.txt

Version 5.5.0 (TBD)
    Added a TypeScript declaration file to the bundle output for TinyMCE core #TINY-3785

>     Added new `table_column_resizing` setting to control how columns are resized when resizing tables using the resize bars #TINY-6001

Reviewer: There's a lot of `resizing` in this sentence. Is that okay?

Author: it is okay, just not pleasant. 
Should `bars` be `handles` ?

Reviewer 2: Could `how columns are resized when resizing tables using the resize bars` maybe be `how table columns are resized when using the resize bars`?

Reviewer: no, they are actual `bars` (horiztonal/vertical lines in the table). It could maybe be `resize bar handles`? That however reads strangely to me.

Author: accepted @metricjs 's suggestion, leaving "bar handles" as just "bars"

>     Added the ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128

Author: ```suggestion
```

    Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011

>     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766

Reviewer 2: I wonder if we should use a better word than `exists`... Not sure what though tbh. This is an existing line though, Idk if we want to change it?

Author: loaded?

Reviewer: We don't want loaded, as it doesn't check that by default. It'll just check if it's configured/enabled

Author: ```suggestion
    Added `hasPlugin` function to the editor API to determine if a plugin is present in the editor configuration #TINY-766
```

Reviewer: It does multiple things (it takes a boolean flag to check if it's just configured or configured and loaded), so this is now maybe misleading? This is probably still better than the original, so I'm okay with it.

    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and the toggle state to be checked #TINY-6032
    Added the ability to delete and navigate HTML media elements without the media plugin #TINY-4211
    Added `fullscreen_native` setting to the fullscreen plugin to enable use of the entire monitor #TINY-6284

>     Added table related oxide variables to the internal style sheets for more granular control over table cell selection appearance #TINY-6311

Reviewer: This is incorrect, `Style API` was correct/what this is normally referred to. It's the skin tool set of variables that allow for tweaking the less files.

Author: ```suggestion
    Added table related oxide variables to the Style API for more granular control over table cell selection appearance #TINY-6311
```

    Added new `toolbar_persist` setting to control the visibility of the inline toolbar #TINY-4847
    Added new APIs to allow for programmatic control of the inline toolbar visibility #TINY-4847
    Added the `origin` property to the `ObjectResized` and `ObjectResizeStart` events, to specify which handle the resize was preformed on #TINY-6242
    Added new StyleSheetLoader `unload` and `unloadAll` APIs to allow loaded stylesheets to be removed #TINY-3926
    Added the `LineHeight` query command and action to the editor #TINY-4843
    Added the `lineheight` toolbar and menu items, and added `lineheight` to the default format menu #TINY-4843
    Added a new `contextmenu_avoid_overlap` setting to allow contextmenus to avoid overlapping matched nodes #TINY-6036

>     Added new `listbox` dialog UI component for rendering a `selectbox` list that accepts nested items #TINY-2236

Reviewer: This change is incorrect, as it's not rendering a `selectbox` list, it's rendering something that is similar.

Author: ```suggestion
    Added new `listbox` dialog UI component for rendering a `selectbox`-styled list that accepts nested items #TINY-2236
```

Reviewer: It's not styled like a selectbox list, it behaves similar to a selectbox list (eg has a dropdown that lets you select a certain value).

Reviewer: It's styled like regular menus to clarify that point, whereas a selectbox's rendering is controlled via the browser.

Reviewer 2: Should we just not mention `selectbox`? It's probably going to confuse people unless we put far more explanation...

Reviewer: We need to describe what it is and a `selectbox`-like component is the easiest.

Author: ```suggestion
    Added new `listbox` dialog UI component for rendering a selection menu that accepts nested items #TINY-2236
```

Author: @lnewson @metricjs :arrow_up:

Reviewer: `selection menu` feels wrong, I can't quite explain why though...

Author: Just "menu" then?

Reviewer: I still prefer the original string than this suggestion sorry.

Reviewer: A menu doesn't make sense for a dialog form component. Can we steal anything from the docs we did for this? https://github.com/tinymce/tinymce-docs/blob/feature/TinyMCE_5.5/ui-components/dialogcomponents.md#listbox

Reviewer: Maybe something like this?
> Added new `listbox` dialog UI component for rendering a menu-like dropdown that allows nested options #TINY-2236

Author: ```suggestion
    Added new listbox dialog UI component for rendering a dropdown that allows nested options #TINY-2236
```

    Added back the ability to use nested items in the `image_class_list`, `link_class_list`, `link_list`, `table_class_list`, `table_cell_class_list`, and `table_row_class_list` settings #TINY-2236
    Changed how CSS manipulates table cells when selecting multiple cells to achieve a semi-transparent selection #TINY-6311
    Changed the `target` property on fired events to use the native event target. The original target for an open shadow root can be obtained using `event.getComposedPath()` #TINY-6128
    Changed the editor to cleanup loaded CSS stylesheets when all editors using the stylesheet have been removed #TINY-3926

>     Changed `imagetools` context menu icon for accessing the `image` settings to use the `image` icon #TINY-4141

Reviewer 2: "`image` settings" -> "`image` dialog"? Or something...

Author: changed to image dialog and dropped code notation.
